Shared/Samples/Create load report/CreateLoadReportView.Views.swift

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension CreateLoadReportView {
+    struct LoadReportView: View {
+        /// The view model for the sample.
+        @ObservedObject var model: Model
+        
+        var body: some View {
+            Form {
+                Section("Phases, Total Customers(C), Total Load(L)") {
+                    ForEach(model.includedPhases.sorted { $0.name < $1.name }, id: \.name) { phase in
```

Comment:
Since the arrays are sorted in `addPhase` and `deletePhase` methods, I think the sort here is not needed.